src/output/file_name.rs

```diff
@@ -201,7 +201,10 @@ impl<'a, 'dir, C: Colours> FileName<'a, 'dir, C> {
         };
 
         if let Some(spaces_count) = spaces_count_opt {
-            let style = iconify_style(self.style());
+            let style = iconify_style(match self.colours.icon() {
+                Some(style) => style,
+                None => self.style(),
+            });
```

Comment:
I know this is nitpick, but couldn't we use an unwrap _or_else here ?

Reply:
Yes, of course, well spotted! ... Done!